src/modules/rover_pos_control/RoverPositionControl.hpp

```diff
@@ -114,6 +115,7 @@ class RoverPositionControl final : public ModuleBase<RoverPositionControl>, publ
 	uORB::Publication<vehicle_attitude_setpoint_s>	_attitude_sp_pub{ORB_ID(vehicle_attitude_setpoint)};
 	uORB::Publication<position_controller_status_s>	_pos_ctrl_status_pub{ORB_ID(position_controller_status)};  /**< navigation capabilities publication */
 	uORB::Publication<actuator_controls_s>		_actuator_controls_pub{ORB_ID(actuator_controls_0)};  /**< actuator controls publication */
+	uORB::Publication<vehicle_local_position_setpoint_s> _local_pos_setpoint_pub{ORB_ID(vehicle_local_position_setpoint)};
```

Comment:
This was added for the sole purpose of enabling PID tuning tab for Velocity / Position controller, as there is no other way to relay the position / velocity information to QGC that is implemented.

We *could implement streaming `vehicle_thrust_setpoint` topic, etc, but for now I wanted to keep the scope of the PR small / rather use a hacky / non-ideal method to get over this issue.